src/lines.rs

```diff
@@ -0,0 +1,96 @@
+use wmidi::Note;
```

Comment:
I’m not sure what the convention is, but I tend to go for singular module names since it reads nicer when a prefix is required. (ie. `line::Note` is nicer than `lines::Note`, IMO)